group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupMetadataManager.java

```diff
@@ -120,6 +121,9 @@
 import static org.apache.kafka.coordinator.group.generic.GenericGroupState.EMPTY;
 import static org.apache.kafka.coordinator.group.generic.GenericGroupState.PREPARING_REBALANCE;
 import static org.apache.kafka.coordinator.group.generic.GenericGroupState.STABLE;
+import static org.apache.kafka.coordinator.group.metrics.GroupCoordinatorMetrics.COMPLETED_REBALANCES_SENSOR_NAME;
```

Comment:
I know we probably can't change the name of the old metric but is it confusing that some generic metrics have the name "generic" and others don't?

Reply:
i agree it's confusing, but not sure if there are alternatives. Would it be better if we rename the sensor variable names at least?

Reply:
Yeah, I wasn't sure if it would be better to just not say "generic" in front, or if that makes it unclear that its the generic group 😅

Reply:
i updated the code so that all new generic group sensors starts with Group instead of Generic Group